frontend/src/Notifications/Notifications.jsx

```diff
@@ -5,11 +5,13 @@ import NotificationCard from './NotificationCard'
 import { WebSocketContext } from '../contexts/WebSocketContext';
 import api from '../api';
 import { useNavigate } from 'react-router-dom'
+import { UserContext } from '../UserContext';
 
 export default function Notifications() {
     const { updatedNotification } = useContext(WebSocketContext);
     const [notifications, setNotifications] = useState([]);
     const navigate = useNavigate()
+    const {user} = useContext(UserContext)
 
     useEffect(() => {
```

Comment:
Hm, is this useEffect called on every render? I don't see a dependency array in it. What is purpose of this hook?